FISH-1654: Log To Console and Verbose mode. (#185)
payara/Payara-Community-Documentation: docs/modules/ROOT/pages/documentation/payara-server/logging/log-to-console.adoc

[[log-to-console]]
= Log To Console

> 

Reviewer: Add a introductory section that explains what Log To Console does.

> The `Log to Console` option is only effective when the Server or the Instance is started with the verbose option.
> 
> [source, shell]
> ----
> asadmin start domain --verbose
> ----
> 
> When the server or domain is not started in verbose mode, the `java.util.ConsoleHandler` will not be active for logging although it is defined as handler in the _logging.properties_ file because there is no console attached with the process running the Payara server or Instance.
> 
> When in verbose mode, the log to Console option will also be active, regardless of the initial value in the _logging.properties_ as otherwise, no log output would be visible on the console. Once started in verbose mode, you can use the Admin Console or the asadmin CLI to change the option.

Reviewer: This sentence could be condensed a little and put under a 'Enabling/Disabling' subheading.

```suggestion
== Enabling
To enable the `Log to Console` option you simply have to start the Server or Instance with the verbose option.

[source, shell]
----
asadmin start domain --verbose
----

When using verbose mode, the log to Console option will also be active, regardless of the initial value in the _logging.properties_  file. Once started in verbose mode, you can use the Admin Console or the asadmin CLI to change this option.

NOTE: If Verbose mode is not used, the `java.util.ConsoleHandler` will not have a console attached to the process and will not be able to log to a console.

=== Disable Log To Console Using The Admin Console

To configure the `Log to Console` option using Admin Console, you can navigate to `Configurations > ${your-config} > Logger Settings > General > Log To File`

image:logging/log_to_file.png[Log to Console disabled]

=== Disable Log To File Using A `asadmin` Command

Use the following command to enable or disabled logging to the console:

[source, shell]
----
asadmin> set-log-attributes com.sun.enterprise.server.logging.GFFileHandler.logtoConsole=false
----
```

The `Log to Console` option controls if the server writes the logging entries to the console. The property only has an effect when the server or instance is started in verbose mode.

[[enable]]
== Enable Logging To Console

To enable the `Log to Console` option you simply have to start the Server or Instance with the verbose option.

[source, shell]
----
asadmin start domain --verbose
----

When the server or domain is not started in verbose mode, the `java.util.ConsoleHandler` will not be active for logging although it is defined as handler in the _logging.properties_ file because there is no console attached with the process running the Payara server or Instance.

When in verbose mode, the log to Console option will also be active, regardless of the initial value in the _logging.properties_ as otherwise, no log output would be visible on the console. Once started in verbose mode, you can use the Admin Console or the asadmin CLI to change the option.

[[using-web-admin-console]]
== Configure Logging To Console Using Admin Console

To configure `Log to Console` option using Admin Console:

image:logging/log_to_file.png[Log to Console disabled]

[[using-asadmin-utility]]

> == Configure Logging To File Using Asadmin CLI

Reviewer 2: I've fixed the typo (File -> Console) in this commit after merging this PR: https://github.com/payara/Payara-Community-Documentation/commit/b6520676f0995a9d55f3562badd5082188199693

[[log-to-console]]
= Log To Console

The `Log to Console` option controls if the server writes the logging entries to the console. The property only has an effect when the server or instance is started in verbose mode.

[[enable]]
== Enable Logging To Console

To enable the `Log to Console` option you simply have to start the Server or Instance with the verbose option.

[source, shell]
----
asadmin start domain --verbose
----

When the server or domain is not started in verbose mode, the `java.util.ConsoleHandler` will not be active for logging although it is defined as handler in the _logging.properties_ file because there is no console attached with the process running the Payara server or Instance.

When in verbose mode, the log to Console option will also be active, regardless of the initial value in the _logging.properties_ as otherwise, no log output would be visible on the console. Once started in verbose mode, you can use the Admin Console or the asadmin CLI to change the option.

[[using-web-admin-console]]

> == Disable Logging To Console Using Admin Console

Reviewer 2: ```suggestion
== Configure Logging To Console Using Admin Console
```

To configure `Log to Console` option using Admin Console:

image:logging/log_to_file.png[Log to Console disabled]

[[using-asadmin-utility]]

> == Disable Logging To File Using Asadmin CLI

Reviewer 2: ```suggestion
== Configure Logging To File Using Asadmin CLI
```

[[log-to-console]]

> = Log To Console

Reviewer 2: ```suggestion
= Logging To Console
```

> = Logging To Console

Reviewer 2: ```suggestion
= Log To Console
```

The `Log to Console` option controls if the server writes the logging entries to the console. The property only has an effect when the server or instance is started in verbose mode.

> == Enabling

Reviewer 2: ```suggestion
[[enable]]
== Enable Logging To Console
```

To enable the `Log to Console` option you simply have to start the Server or Instance with the verbose option.

[source, shell]
----
asadmin start domain --verbose
----

When the server or domain is not started in verbose mode, the `java.util.ConsoleHandler` will not be active for logging although it is defined as handler in the _logging.properties_ file because there is no console attached with the process running the Payara server or Instance.

When in verbose mode, the log to Console option will also be active, regardless of the initial value in the _logging.properties_ as otherwise, no log output would be visible on the console. Once started in verbose mode, you can use the Admin Console or the asadmin CLI to change the option.

[[using-web-admin-console]]

> == Using the Admin Console

Reviewer 2: ```suggestion
== Disable Logging To Console Using Admin Console
```

To configure `Log to Console` option using Admin Console:

image:logging/log_to_file.png[Log to Console disabled]

[[using-asadmin-utility]]

> == Using an `asadmin` command

Reviewer 2: ```suggestion
== Disable Logging To File Using Asadmin CLI
```
Current name is Asadmin CLI: https://payara.atlassian.net/wiki/spaces/MARK/pages/2387247181/Feature+Re-naming
